producer/ue_authentication.go

```diff
@@ -222,7 +221,6 @@ func UeAuthPostRequestProcedure(updateAuthenticationInfo models.AuthenticationIn
 
 		var eapPkt radius.EapPacket
 		var randIdentifier int
-		rand.Seed(time.Now().Unix())
 
 		eapPkt.Code = radius.EapCode(1)
 		randIdentifier = rand.Intn(256)
```

Comment:
If you agree with this solution, please run all the necessary tests (including an end-to-end test) to make sure everything is working as expected
```go
		src := rand.NewSource(time.Now().UnixNano())
		random := rand.New(src)

		eapPkt.Code = radius.EapCode(1)
		randIdentifier = random.Intn(256)
```

Reply:
I provided a solution that uses the more secure crypto/rand (instead of math/rand). Now for tests I added 1 here but there were 0 tests in this project at all which doesn't help validating that we didn't break anything. I'll also run an end-to-end test to validate.

Reply:
I just validated this in an end-to-end test with gnbsim and it passed.

Reply:
> I just validated this in an end-to-end test with gnbsim and it passed.

Great, thanks!